src/python/WMCore/Services/AlertManager/AlertManagerAPI.py

```diff
@@ -0,0 +1,94 @@
+"""
+
+AlertManagerAPI - send alerts to MONIT AlertManager via API calls
+"""
+
+from __future__ import division
+from datetime import timedelta, datetime
+import socket
+import json
+import logging
+
+from WMCore.Services.pycurl_manager import RequestHandler
+from Utils.Timers import LocalTimezone
+
+
+class AlertManagerAPI(object):
+    """
+    A class used to send alerts via the MONIT AlertManager API
+    """
+
+    def __init__(self, alertManagerUrl):
+        self.alertManagerUrl = alertManagerUrl
+        # sender's hostname is added as an annotation
+        self.hostname = socket.gethostname()
+        self.mgr = RequestHandler()
+        self.ltz = LocalTimezone()
```

Comment:
How about creating another object attribute `self.generatorURL` with the alertmanager front page as default? Then on the `send` method, we default that to None, and if no value is provided, we fallback to the dafault value in the instance.

Reply:
I'm not sure about setting `generatorURL` to the AlertManager page. Unless this is sent in the body of the email, it would just be a link to the AlertManager page that is clickable from the AlertManager page.

Reply:
I assume this `generatorURL` is optional then? We can simply pass the default empty string from the code below and everything would work, right?

Reply:
Yes, I tested with a blank string and things work. Alert shows up with no "Source" on the AM page.